_posts/2024-10-15-anaconda-defaults.md

```diff
@@ -1,47 +1,53 @@
----
+That's it! 
 layout: post
-title: Guide to Ananconda `defaults` channel
+title: A Guide to the Anaconda `defaults` channel
 
 tags:
-  - anaconda
-  - conda
+  - Conda
   - bioconda
-  - software
+  - conda-forge
+  - defaults
 contributors:
   - ammarziz
 reviewers:
-- TBA
+- pmenzel
+- jfy133
 ---
 
-# Guide to Anaconda `defaults` channel
+# A Guide to the Anaconda `defaults` channel
+
+![In the art style of 1990s Disney, cel shading: A mysterious blacksmith in a dark, forge-like setting. She wears a sleek, gold-plated helmet embossed with the symbol of a snake and an armored robe with intricate designs. Sparks fly as she hammers a molten weapon on an anvil, which begins to take the shape of a coiled snake. The forge’s flames cast a warm glow, reflecting off her helmet and tools. The snake is a python with a green sheen.]({% link assets/images/way-of-the-mambalorian.webp %})
```

Comment:
could you also add a `image: /assets/images/way-of-the-mambalorian.webp` to have this used in the social media preview as well?

Reply:
Done!